### PR TITLE
Add JSBSim Support

### PR DESCRIPTION
**Pull request by owsorber**
Add important files from https://github.com/AOS55/Fixedwing-Airsim/tree/master for jsbsim support

### CODE REVIEW

File: simulation/jsbsim_simulator.py [hunk 1]
```diff
@@ -0,0 +1,308 @@
...
+    set_throttle_mixture_controls()
+        sets aircraft mixture and throttle controls
+    raise_landing_gear()
+        raises the aircraft's landing gear
+    """
+
+    encoding = 'utf-8'
+
+    # Root directory of jsbsim on the computer used for simulation
+    ROOT_DIR = os.path.abspath(r"/Users/owensorber/opt/anaconda3/lib/python3.8/site-packages/jsbsim")
```
Comment (owsorber) [inline simulation/jsbsim_simulator.py:78]: Update to path on eric's computer @yekime

File: simulation/jsbsim_simulator.py [hunk 2]
```diff
@@ -0,0 +1,308 @@
...
+
+        :return: JSBSim model name
+        """
+        name: str = self.fdm.get_model_name().decode(self.encoding)
+        if name:
+            return name
+        else:
+            return None
+
+    def initialise(self, dt: float, model_name: str, init_conditions: Dict['prp.Property', float] = None) -> None:
```
Comment (owsorber) [inline simulation/jsbsim_simulator.py:132]: Might need to come up with a way to initialize randomly from a reset distribution (not necessarily from a basic takeoff file) as an exploration technique

**Comment by owsorber** [comment]
@yekime add the updated basic_ic.xml file for fixed wing to the simulation directory

File: assets/x8.xml [hunk 3]
```diff
@@ -0,0 +1 @@
+<!-- Replace with final version of our fixed-wing model -->
```
Comment (owsorber) [inline assets/x8.xml:1]: @yekime update with current fixed wing model
and if there are any other asset files related to sim/unreal might be useful to store in this folder

**Comment by owsorber** [comment]
Should also add one more file to simulate one flight modeled off of [main.py in Fixedwing-Airsim](https://github.com/AOS55/Fixedwing-Airsim/blob/master/src/main.py)

**Comment by yekime** [comment]
lgtm

**Pull request merged**